source/macos/AdaptiveCards/AdaptiveCards/Views/ACRInputErrorTextField.swift

```diff
@@ -17,8 +17,14 @@ class ACRInputErrorTextField: ACRInputLabelTextField {
     
     override func setAttributedString() {
         guard let errorMessage = inputElement.getErrorMessage(), !errorMessage.isEmpty else { return }
-        let attributedErrorMessageString = NSMutableAttributedString(string: errorMessage)
         let errorStateConfig = renderConfig.inputFieldConfig.errorStateConfig
+        
+        let attachment = NSTextAttachment()
+        attachment.image = BundleUtils.getImage("warning-badge-filled", ofType: "png")
+        attachment.setImageBounds(from: errorStateConfig.font)
```

Comment:
Like we do for calendar, first we need to see if image available in render config, if not use the default image